### ISSUE TITLE
Implement memory "save/load" (Chipeit)

### ISSUE THREAD

**Issue by mikelcaz** [SergioLuis/CHIPEIT#38]
## Assigned people

**Developer**:
**Reviewer**:

---

## Description

In order to save/load a game state, is necessary to save/load the main memory. The API must be extended somehow to achieve that.

Saving the memory implies a runtime behavior, but loading it doesn't as long a new Chipeit object can be generated.

---

**Comment by SergioLuis** [comment]
I wanted to do #39 on a separate task because #44 depends on #39, but not on this one. And we can have game previews (#44) without having to implement the grid interface (#37) first.
